src/app/components/Video/index.test.jsx

```diff
@@ -5,7 +5,7 @@ import Video from './index';
 describe('Video', () => {
   describe('with data', () => {
     const props = {
-      pid: 'p00a0a0m',
+      id: 'mp#p00a0a0m',
       kind: 'episode',
       title: 'Hello World',
       items: [
```

Comment:
This test example doesn't have all of the required props - e.g. the mediaPlayerSettings. Could you please update it so that it has all the required props?